constellations/gemini-admin/src/emails/accountInfo.js

```diff
@@ -0,0 +1,134 @@
+                    email.splice(email.indexOf(user.email), 1);
+                    this.setState({
+                        selectedUserEmails: email,
+                    });
+                }
+            });
+        }
+    };
+
+    render = () => {
+        const id = this.props.id;
+        const email = this.props.email;
+        const users = this.props.users;
+        const transactions = this.props.transactions;
+        const name = this.props.name;
+
+        var emails = [];
+        users.map((user, index) => {
+            emails.push(user.email);
+        });
+
+        const userRows = users.map((user, index) => {
+            var status = user.isGuardian ? "(guardian" : "(student";
+            status += user.email == email ? ", primary contact)" : ")";
+
+            if (this.state.selectedUserEmails.length > 0) {
+                this.state.selectedUserEmails.map((existingEmail, index) => {
+                    if (emails.indexOf(existingEmail) == -1) {
+                        this.setState({
+                            selectedUserEmails: [],
+                        });
+                    }
+                });
+            }
+            return (
+                <div className="row" key={index}>
+                    <input
+                        type="checkbox"
+                        onChange={(e) => this.onCheckUser(e, user.id)}
+                    />
+                    <span className="column">{getFullName(user)}</span>
+                    <span className="column">{user.email}</span>
+                    <span className="column status">{status}</span>
+                </div>
+            );
+        });
+
+        var balance = 0;
+        transactions.map((transaction, index) => {
+            const amount = transaction.amount;
+            balance += parseInt(amount);
+        });
+
+        return (
+            <section id="account-info">
+                <span id="account-number">Account Information</span>
+                <h2>AccountId: {id}</h2>
+                <h2>Primary Email: {email}</h2>
+                <div id="account-users">
+                    <h2>Users in Account</h2>
+                    {userRows}
+                </div>
+                <div id="transaction-footer">
+                    <span>
+                        Account Balance: <b>{this.formatCurrency(balance)}</b>
+                    </span>
+                </div>
+                <div className="email-template">
+                    <span id="template-title">Generated Email Template</span>
+                    <h3>To: {this.state.selectedUserEmails.toString()}</h3>
+                    <h3>
+                        Subject: Math Navigator: Account Balance Payment
+                        Reminder
+                    </h3>
+                    <h3>Message: </h3>
+                    <div className="generated-email">
+                        <p>Hello {name},</p>
```

Comment:
call this `primaryContactName` to ensure we're always talking to the person who's the primary contact / owner of the account.